src/governance/README.md

```diff
@@ -0,0 +1,428 @@
+# Governance
+
+<p style="background: #f8d7da; color: #721c24; padding: 5px">
+<strong>IMPORTANT</strong> This document is adapted from
+<a href="https://rust-lang.github.io/rfcs/1068-rust-governance.html">RFC 1068</a>
+and is currently being actively worked on, however there may be large parts of
+Rust's governance that are missing, incomplete, or out of date.
+</p>
+
+## Core team
+
+**The core team serves as leadership for the Rust project as a whole**. In
+particular, it:
+
+- **Sets the overall direction and vision for the project.** That means setting
+  the core values that are used when making decisions about technical
+  tradeoffs. It means steering the project toward specific use cases where Rust
+  can have a major impact. It means leading the discussion, and writing RFCs
+  for, _major_ initiatives in the project.
+
+- **Sets the priorities and release schedule.** Design bandwidth is limited, and
+  it's dangerous to try to grow the language too quickly; the core team makes
+  some difficult decisions about which areas to prioritize for new design, based
```

Comment:
The compiler, libs, and lang teams decide what design and implementation areas to focus on autonomously.